packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -933,6 +942,15 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
         }
         return gas;
     },
+    [ERC20BridgeSource.KyberDmm]: (fillData?: FillData) => {
+        // TODO: Different base cost if to/from ETH.
+        let gas = 95e3;
+        const path = (fillData as UniswapV2FillData).tokenAddressPath;
+        if (path.length > 2) {
+            gas += (path.length - 2) * 65e3; // +65k for each hop.
+        }
```

Comment:
slightly higher gas here bc the Kyber DMM router performs this validation for each pool: https://github.com/dynamic-amm/smart-contracts/blob/master/contracts/periphery/DMMRouter02.sol#L629-L636